### PR TITLE
Implement mentor availability endpoint

### PR DESCRIPTION
**Pull request by Shrenik0321**
## Purpose
The purpose of this endpoint is for the mentor to update their availability

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [ ] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by Shrenik0321** [comment]
I havent written the tests yet. Wanted to make sure the endpoint is fine. Will do once approved. Also im not sure about the route @anjula-sack

File: src/routes/profile/profile.route.ts [hunk 1]
```diff
@@ -4,12 +4,18 @@ import {
...
 const profileRouter = express.Router()
 
 profileRouter.get('/profile', requireAuth, getProfileHandler)
 profileRouter.put('/profile', requireAuth, updateProfileHandler)
 profileRouter.delete('/profile', requireAuth, deleteProfileHandler)
+profileRouter.put(
+  '/mentor/availability',
+  requireAuth,
+  mentorAvailabilityHandler
+)
```
Comment (anjula-sack) [inline src/routes/profile/profile.route.ts:19]: This should go to mentor routes @Shrenik0321

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: src/routes/admin/admin.route.test.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+describe('Get all users route', () => {
+  beforeAll(async () => {
+    server = await startServer(port)
+    agent = supertest.agent(server)
+    adminAgent = supertest.agent(server)
+
+    const defaultUser = {
+      email: `test${randomString}@gmail.com`,
+      password: '123'
+    }
```
Comment (anjula-sack) [inline src/routes/admin/admin.route.test.ts:26]: You can import this from mock.ts.

File: src/routes/admin/admin.route.test.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+import supertest from 'supertest'
+import Profile from '../../entities/profile.entity'
+import { ProfileTypes } from '../../enums'
+import { dataSource } from '../../configs/dbConfig'
+import bcrypt from 'bcrypt'
+
+const port = Math.floor(Math.random() * (9999 - 3000 + 1)) + 3000
+
+const randomString = Math.random().toString(36)
+const randomStringAdmin = Math.random().toString(36)
```
Comment (anjula-sack) [inline src/routes/admin/admin.route.test.ts:12]: This can be removed. Check mock.ts

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Review by anjula-sack — APPROVED** [review]
Thank you! @Shrenik0321

**Pull request merged**